e2e/tests/nightly/admin/institution-admin-ui.spec.ts

```diff
@@ -50,6 +50,8 @@ describe('Institution Admin', () => {
     expect(await institutionEditPage.getCTEmailAcceptedValue()).toBe(AcceptedAddressSelectValue.Domains);
     // verify that the Accepted Email Domains text area is present in controlled-card-details
     institutionEditPage.getCTEmailDomainsInput();
+    //verify that the save button is disabled
+    expect(await institutionEditPage.getSaveButton().isCursorNotAllowed()).toBe(true);
```

Comment:
@jmthibault79 I just tried debugging the SAVE button issue. Seems like it could be resolved if the button label text is capitalized instead of uppercase. (Save vs SAVE)

Reply:
I don't think it matters.  OK to look for "SAVE"